src/main/java/org/flickit/flickitassessmentcore/adapter/out/persistence/assessment/AssessmentJpaRepository.java

```diff
@@ -18,11 +18,11 @@ public interface AssessmentJpaRepository extends JpaRepository<AssessmentJpaEnti
         "LEFT JOIN AssessmentResultJpaEntity r " +
         "ON a.id = r.assessment.id " +
         "WHERE a.spaceId IN :spaceIds AND " +
-        "a.deletionTime=:deletionTime AND " +
+        "a.deleted=:deleted AND " +
         "(a.assessmentKitId=:kitId OR :kitId IS NULL) AND " +
         "r.lastModificationTime = (SELECT MAX(ar.lastModificationTime) FROM AssessmentResultJpaEntity ar WHERE ar.assessment.id = a.id) " +
         "ORDER BY a.lastModificationTime DESC")
-    Page<AssessmentListItemView> findBySpaceIdOrderByLastModificationTimeDesc(List<Long> spaceIds, Long kitId, long deletionTime, Pageable pageable);
+    Page<AssessmentListItemView> findBySpaceIdOrderByLastModificationTimeDesc(List<Long> spaceIds, Long kitId, boolean deleted, Pageable pageable);
```

Comment:
Remove deleted parameter and change method name to `findBySpaceIdAndDeletedFalseOrderByLastModificationTimeDesc`